__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+const testData = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+
+
+afterAll( () => {
+    if (db.end) db.end();
+});
+
+beforeEach( () => seed(testData) );
+
+
+    
+
+
+
+
+describe("GET /api/categories", () => {
+    test("Responds with an array of category objects, each of which should have the properies of 'slug' and 'description'.", () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then(( {body} ) => {
+                const {categories} = body;
+                expect(categories).toBeInstanceOf(Array);
+                expect(categories).toHaveLength(4);
+                categories.forEach((category) => {
+                    expect(category).toEqual(
+                        expect.objectContaining({
+                            slug: expect.any(String),
+                            description: expect.any(String)
+                        })
```

Comment:
Good test, nice to see your checking the structure and design of the data coming back, to see it's what you expect it to look like